scripts/upgrade_disk_storage_holesky.sh

```diff
@@ -0,0 +1,236 @@
+            fi
+        else
+            COMPOSE_FILE="${DOCKER_DIR}/${el_node_type}-holesky-0${i}/docker-compose.yml"
+            if [[ -f "$COMPOSE_FILE" ]]; then
+                echo "Stopping containers for $COMPOSE_FILE..."
+                docker compose -f "$COMPOSE_FILE" down || echo "Failed to stop containers for $COMPOSE_FILE"
+            else
+                echo "Compose file not found for $COMPOSE_FILE"
+            fi
+        fi
+    done
+}
+
+backup_docker_data() {
+    local el_node_type=$1
+    echo "Creating backup directory: $CONFIG_BACKUP_DIR"
+    mkdir -p "$CONFIG_BACKUP_DIR"
+
+    if [[ "$el_node_type" == "erigon" ]]; then
+        rsync -av --info=progress2 \
+        --exclude "${DOCKER_DIR}/${el_node_type}-holesky-01/data" \
+        --exclude "${DOCKER_DIR}/${el_node_type}-holesky-02/data" \
+        --exclude "${DOCKER_DIR}/${el_node_type}-holesky-03/data" \
+        --exclude "${DOCKER_DIR}/${el_node_type}-holesky-04/data" \
+        "$DOCKER_DIR/" "$CONFIG_BACKUP_DIR/"
+    else
+        rsync -av --info=progress2 \
+        --exclude "${DOCKER_DIR}/${el_node_type}-holesky-01/node/data" \
+        --exclude "${DOCKER_DIR}/${el_node_type}-holesky-02/node/data" \
+        --exclude "${DOCKER_DIR}/${el_node_type}-holesky-03/node/data" \
+        --exclude "${DOCKER_DIR}/${el_node_type}-holesky-04/node/data" \
+        "$DOCKER_DIR/" "$CONFIG_BACKUP_DIR/"
+    fi
+    echo "Synced config data to backup directory"
+
+    if [[ "$el_node_type" == "erigon" ]]; then
+        echo "Creating backup directory for chaindata: $CHAINDATA_BACKUP_DIR"
+        mkdir -p "$CHAINDATA_BACKUP_DIR"
+
+        echo "Creating backup directory for snapshots: $SNAPSHOTS_BACKUP_DIR"
+        mkdir -p "$SNAPSHOTS_BACKUP_DIR"
+
+        rsync -av --info=progress2 \
+        "$DOCKER_DIR/${el_node_type}-holesky-01/node/data/chaindata" "$CHAINDATA_BACKUP_DIR/"
+
+        rsync -av --info=progress2 \
+        "$DOCKER_DIR/${el_node_type}-holesky-01/node/data/snapshots" "$SNAPSHOTS_BACKUP_DIR/"
+    elif [[ "$el_node_type" == "geth" ]]; then
+        echo "Creating backup directory for chaindata: $CHAINDATA_BACKUP_DIR"
+        mkdir -p "$CHAINDATA_BACKUP_DIR"
+
+        rsync -av --info=progress2 \
+        "$DOCKER_DIR/${el_node_type}-holesky-01/node/data/geth/chaindata" "$CHAINDATA_BACKUP_DIR/"
+    elif [[ "$el_node_type" == "nethermind" ]]; then
+        echo "Creating backup directory for chaindata: $CHAINDATA_BACKUP_DIR"
+        mkdir -p "$CHAINDATA_BACKUP_DIR"
+        rsync -av --info=progress2 \
+        "$DOCKER_DIR/${el_node_type}-holesky-01/node/data/nethermind_db" "$CHAINDATA_BACKUP_DIR/"
+    fi
+    echo "Synced chain data to backup directory"
+}
+
+setup_raid_and_mount() {
+    # Unmount the devices
+    echo "Unmounting devices..."
+    sudo umount /docker
+    sudo umount /mnt/sdc
+
+    # Wipe the devices
+    echo "Wiping devices..."
+    sudo wipefs -a /dev/sdc
+    sudo wipefs -a /dev/sdd
```

Comment:
Doing stuff like this is a dangerous game if names of devices are different on a host :D.